cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
OK, I could do that

On Fri, Apr 8, 2016 at 07:08, Misagh Moayyed <notifications@github.com> wrote:
In cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml [https://github.com/Jasig/cas/pull/1676#discussion_r59010017] :

> @@ -0,0 +1,29 @@
> +<?xml version="1.0" encoding="UTF-8"?>
> +<beans xmlns="http://www.springframework.org/schema/beans"
> +       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"


I would feel better if you moved the monitor list out and back into the deployer. That is something a deployer would need to set more often the aliases, and is not so much about mapping aliases.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/Jasig/cas/pull/1676/files/9c8a4b49aee43d91c0113ec6f61045218cf9e71f#r59010017]